nucypher/network/server.py

```diff
@@ -181,10 +182,13 @@ def learn_about_announced_nodes():
                     message = f"Suspicious Activity: Discovered node with bad signature: {node}.  Announced via REST."
                     log.warn(message)
                     this_node.suspicious_activities_witnessed['vladimirs'].append(node)
-
+                except NodeSeemsToBeDown as e:
+                    # This is a rather odd situation - this node *just* contacted us and asked to be verified.  Where'd it go?  Maybe a NAT problem?
+                    log.info(f"Node announced itself to us just now, but seems to be down: {node}.  Response was {e}.")
+                    log.debug(f"Phantom node certificate: {node.certificate}")
                 # Async Sentinel
                 except Exception as e:
-                    log.critical(str(e))
+                    log.critical(f"This exception really needs to be handled differently: {e}")
```

Comment:
OK..... haha - Perhaps an Issue #?

Reply:
It's not a *specific* issue; it's just that *any* exception that makes it this far needs to be handled differently.